lib/screens/home_page.dart

```diff
@@ -162,6 +168,17 @@ class _HomePageState extends State<HomePage> {
                     ),
                   ),
                 ),
+                c.Padding(
```

Comment:
Use Material based `Scaffold`, to position Floating button

Reply:
When I used that there was render overflow

Reply:
ok i will do that